src/transformers/integrations.py

```diff
@@ -878,6 +878,17 @@ def on_train_end(self, args, state, control, **kwargs):
             if self._auto_end_run and self._ml_flow.active_run():
                 self._ml_flow.end_run()
 
+    def on_save(self, args, state, control, **kwargs):
+        if self._initialized and state.is_world_process_zero and self._log_artifacts:
```

Comment:
I think this new behavior (uploading the saved model at each save) should be documented above with the `HF_MLFLOW_LOG_ARTIFACTS` env variable.

Reply:
fixed the documentation - thanks for your review!